lib/spdy-transport/connection.js

```diff
@@ -185,6 +197,13 @@ Connection.prototype._init = function init() {
     self.setTimeout(0);
   });
 
+  state.window.recv.on('overflow', function() {
```

Comment:
Sorry for nit-picking, but I think that this function may be put right above these two `on()` calls, and reused between them.